set.mm

```diff
@@ -148551,6 +149801,20 @@ Splicing words (substring replacement)
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  $c repeatS $.
+
+  $( Extend class notation with words consisting of one repeated symbol. $)
+  creps $a class repeatS $.
+
+  ${
+    $d n s x $.
+    $( Definition to construct a word consisting of one repeated symbol, often
+       called "repeated symbol word" for short in the following.  (Contributed
+       by Alexander van der Vekens, 4-Nov-2018.) $)
+    df-reps $a |- repeatS
+                 = ( s e. _V , n e. NN0 |-> ( x e. ( 0 ..^ n ) |-> s ) ) $.
```

Comment:
It looks more and more specialized. I hope there are (future?) applications to justify these definitions.

Reply:
This definition is not new, also only moved to the corresponding subsection. It is used to prove the friendship theorem.